### PR TITLE
Try merging the projects

### PR DESCRIPTION
**Pull request by avik-pal**

### CODE REVIEW

File: src/SciMLBenchmarks.jl [hunk 1]
```diff
@@ -61,17 +69,20 @@ end
...
+      @eval @subprocess begin
+        using SciMLBenchmarks
+        SciMLBenchmarks.weave_file($folder, $file, $build_list)
       end
+    catch e
+      @show folder, file
+      @error(e)
+    end
   end
 end
```
Comment (avik-pal) [inline src/SciMLBenchmarks.jl:87]: @ChrisRackauckas What do you think about this? This prevents segfaults at least locally. All the subprocesses are blocking and carry over all the options so they won't interfere with the benchamrks

Comment (ChrisRackauckas) [reply src/SciMLBenchmarks.jl:87]: That looks fine, just neds a rebase

File: benchmarks/NonlinearProblem/nonlinear_solver_23_tests.jmd [hunk 2]
```diff
@@ -100,10 +100,10 @@ function benchmark_problem!(prob_name; solver_tracker=solver_tracker, selected_N
     solvers_TR = filter(s -> s.type==:TR, successful_solvers)
     solvers_LM = filter(s -> s.type==:LM, successful_solvers)
     solvers_PT = filter(s -> s.type==:PT, successful_solvers)
```
Comment (avik-pal) [inline benchmarks/NonlinearProblem/nonlinear_solver_23_tests.jmd:102]: @yonatanwesen why was this added as a separate solver list? It is a single solver so we don't need a separate plot. Can you open a follow up PR to fix this

**Comment by avik-pal** [comment]
@ChrisRackauckas  this should be good to go

**Pull request merged**